src/appservice-kube/azext_appservice_kube/_params.py

```diff
@@ -37,6 +37,23 @@ def load_arguments(self, _):
                    completer=get_resource_name_completion_list('Microsoft.Web/sites'), id_part='name',
                    help="name of the web app. You can configure the default using `az configure --defaults web=<name>`")
 
+    with self.argument_context('webapp update') as c:
+        c.argument('client_affinity_enabled', help="Enables sending session affinity cookies.",
+                   arg_type=get_three_state_flag(return_label=True))
+        c.argument('https_only', help="Redirect all traffic made to an app using HTTP to HTTPS.",
+                   arg_type=get_three_state_flag(return_label=True))
+        c.argument('force_dns_registration', help="If true, web app hostname is force registered with DNS",
```

Comment:
if we are redefining these - lets remove this support in extension - since I don't believe these properties are supported on backend anymore :-/